blueprints/validator/files/app/validators/__name__.js

```diff
@@ -0,0 +1,35 @@
+export default function validate<%= classifiedModuleName %>(options = {}) {
+  return (key, newValue, oldValue, changes) => {
+    /*
```

Comment:
Don't need this comment block, it is going to be generated for every validator which can be quite annoying. The readme already explains how to implement a validator